nagios/config/wavefront-csp-token.cfg

```diff
@@ -0,0 +1,41 @@
+# Nagios command definitions for host and service metrics to the Wavefront server using CSP API Token
```

Comment:
Nagios command definitions for host and service metrics to the Operations for Applications server using a VMware Cloud services API token